rotkehlchen/rotkehlchen.py

```diff
@@ -806,7 +807,9 @@ def query_balances(
 
         # retrieve loopring balances if module is activated
         if self.chain_manager.get_module('loopring'):
-            balances[str(Location.LOOPRING)] = self.chain_manager.get_loopring_balances()
+            loopring_balances = self.chain_manager.get_loopring_balances()
+            if len(loopring_balances):
```

Comment:
explicit. Never compare 0 to a Falsy value. This is a source of many many bugs.

```suggestion
            if len(loopring_balances) != 0:
```